packages/devui-vue/devui-cli/commands/generate-dts.js

```diff
@@ -19,11 +19,11 @@ function generateIndexDts(buildDir) {
 
 exports.generateDts = () => {
   generateIndexDts(outputDir);
-
+  const availableTargetFiles = fs.readdirSync(outputDir);
```

Comment:
```suggestion
  for(const name of components) {
    const destDts = path.resolve(outputDir, `${name}/index.d.ts`);
    fs.copyFile(srcDts, destDts, (err) => {
      if (err) {
        console.log(err);
        logger.error(`拷贝组件${name}的ts类型文件失败！`);
      }
    });
  }
```
打印了一下拷贝错误日志，发现是因为目标目录不存在，所以将不存在的目标目录过滤掉，解决报错。